docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -87,29 +87,33 @@ Allows you to select which tests should be run.
 filter(callback) → this
 ```
 
-Parameter  | Type                                           | Description
----------- | ---------------------------------------------- | ----------------------------------------------------------------
-`callback` | `function(testName, fixtureName, fixturePath)` | The callback that determines if a particular test should be run.
+Parameter  | Type                                                                  | Description
+---------- | --------------------------------------------------------------------- | ----------------------------------------------------------------
+`callback` | `function(testName, fixtureName, fixturePath, testMeta, fixtureMeta)` | The callback that determines if a particular test should be run.
 
 The callback function is called for each test in the files specified using the [src](#src) method.
```

Comment:
The callback function is called for each test in the files the [src](#src) method specifies.